.github/workflows/basic.yml

```diff
@@ -133,6 +133,7 @@ jobs:
         -DCMAKE_C_COMPILER=${{matrix.compiler.c}}
         -DCMAKE_CXX_COMPILER=${{matrix.compiler.cxx}}
         -DUMF_BUILD_LEVEL_ZERO_PROVIDER=${{matrix.level_zero_provider}}
```

Comment:
you could add a similar test to the L0 prov. (one build without cuda_prov enabled)

Reply:
done